Asin (#1774)
Codecademy/docs: content/c-sharp/concepts/math-functions/terms/asin/asin.md

---
Title: '.Asin()'
Description: 'Returns an angle in radians whose sine is a specified number.'
Subjects: 
  - 'Computer Science'
Tags:
  - 'Libraries'
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

> The **Math.Asin()** function computes an angle in radians whose sine is a specified number. In Math, this is known as the inverse sine function.

Reviewer: ```suggestion
The **Math.Asin()** method computes an angle in radians whose sine is a specified number. In Math, this is known as the inverse sine function.
```

> The **Math.Asin()** function computes an **angle** in **radians** whose **sine** is a specified number. In Math, this is known as the **inverse sine** function.

Reviewer 2: ```suggestion
The **Math.Asin()** function computes an angle in radians whose sine is a specified number. In Math, this is known as the inverse sine function.
```

Author: @SSwiniarski have commited the changes as suggested, rgds

---
Title: '.Asin()'
Description: 'Returns an angle in radians whose sine is a specified number.'
Subjects: 
  - 'Computer Science'
Tags:
  - 'Libraries'
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

The **Math.Asin()** method computes an angle in radians whose sine is a specified number. In Math, this is known as the inverse sine function.

## Syntax

```pseudo

> Math.Asin(value);

Reviewer: missing closing backticks

`value`: the input value of type double, which must be in the range -1 <= `value` <= 1.

Returns an angle measured in radians, of type `double`, whose sine is `value`. If an invalid value is passed to the function, or no value is passed at all, `NaN` is returned.

## Codebyte Example

```cs
using System;

> class CodeacademyDemo

Reviewer: ```suggestion
class CodecademyDemo
```

---
Title: '.Asin()'

> Description: 'Math function which returns the inverse sine of an angle'

Reviewer 2: ```suggestion
Description: 'Returns an angle in radians whose sine is a specified number.'
```

Subjects: 
  - 'Computer Science'
Tags:
  - 'Libraries'
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

The **Math.Asin()** function computes an **angle** in **radians** whose **sine** is a specified number. In Math, this is known as the **inverse sine** function.

## Syntax

```public static double Asin (double value)```

> ## Parameters

Reviewer 2: Remove this header.

> ```value```: the input value, which must be in the range -1 <= value <= 1

Reviewer 2: ```suggestion
`value` is the input value, which must be in the range -1 <= `value` <= 1
```

> ## Return value

Reviewer 2: Remove this header.

> An angle measured in radians, and of type ```double```, whose sine is ```value```. If an invalid value is passed to the function, or no value is passed at all, ```NaN``` is returned.

Reviewer 2: ```suggestion
This returns an angle measured in radians, of type `double`, whose sine is `value`. If an invalid value is passed to the function, or no value is passed at all, `NaN` is returned.
```
